src/olympia/search/indexers.py

```diff
@@ -70,12 +72,21 @@ def create_new_index(index_name=None):
     if index_name is None:
         index_name = BaseSearchIndexer.get_index_alias()
 
+    index_settings = INDEX_SETTINGS.copy()
```

Comment:
Need to be a `deepcopy()`, otherwise you might change `INDEX_SETTINGS` as well.

Reply:
fair point, I'll change that.